MapboxNavigation/NavigationMapView.swift

```diff
@@ -359,12 +359,12 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     open var tracksUserCourse: Bool = false {
         didSet {
             if tracksUserCourse {
-                enableFrameByFrameCourseViewTracking(for: 3)
+                enableFrameByFrameCourseViewTracking(for: 2)
```

Comment:
Just curious what this change means

Reply:
There's no way to figure out when a gesture finishes with velocity so this enables frame by frame tracking for 2 seconds which means that the user puck will update on every `-[MGLMapView(MGLNavigationAdditions) mapViewDidFinishRenderingFrameFullyRendered:]`. We might be able to simplify this logic when https://github.com/mapbox/mapbox-gl-native/pull/11151 lands.

Reply:
One approach would be to implement `mapView:shouldChangeFrom:to:` to capture when a camera change occurs due to a gesture (since that method only gets called in response to gestures) or set that state in a completion handler; for non-gesture-related camera changes, the velocity animation finishes when `mapView:regionDidChange:animated:` is called.